embc-app/Controllers/RegistrationsController.cs

```diff
@@ -68,6 +68,11 @@ public async Task<IActionResult> Create([FromBody] ViewModels.Registration item)
                 return BadRequest(ModelState);
             }
 
+            item.CompletedBy = new ViewModels.Volunteer
+            {
+                Externaluseridentifier = httpContextAccessor?.HttpContext?.User?.FindFirstValue(EssClaimTypes.USER_ID)
```

Comment:
If you look at ViewModels/Registration.cs, line 18, you'll notice the CompletedById is mapped from the Id property in the view model